features/addons/src/main/feature/feature.xml

```diff
@@ -3,10 +3,13 @@
 
 	<!-- these are 2.x add-ons in separate repositories (which do not have their own Karaf feature defined), so we include them here -->
 
-    <feature name="openhab-binding-zigbee" description="ZigBee Binding" version="${project.version}">
+    <feature name=“openhab-binding-zigbee-ember" description="ZigBee Binding for Ember NCP" version="${project.version}">
```

Comment:
With respect to the Github diff view I assume the quote character in front of openHAB is not the one that has been expected.